keps/sig-scheduling/4815-dra-partitionable-devices/README.md

```diff
@@ -394,8 +411,17 @@ part of the multi-host device. But it can not guarantee that all pods will be
 scheduled, since pods will be subject to any other constraints (like sufficient
 CPU and memory) during scheduling.
 
-A better story should be in place for beta, including a plan for alignment and
-possible integration with Kueue.
+These challenges are being addressed as part of the gang scheduling effort that
+also includes alignment with DRA.
+
+#### Validation moved from admission to runtime
+
+By allowing cross-`ResourceSlice` references, we will not be able to validate
+that the references can actually be resolved during admission since we are only
+able to validate a `ResourceSlice` in isolation. This means that users will not
+discover mistakes until a `ResourceClaim` actually tries to allocate a device
+that belongs to the `ResourcePool`. This means more complexity and a less user-friendly
```

Comment:
Two things:

1) ResourceSlice is not a human-created resource in the typical case. So, errors here would be bugs in the driver.

2) We can have a controller that surfaces inconsistent pools / errors in a ResourceSlice's status, rather than waiting until scheduling time.